toast-tk-runtime/src/main/java/io/toast/tk/runtime/mail/SmtpConfigProvider.java

```diff
@@ -0,0 +1,114 @@
+package io.toast.tk.runtime.mail;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.security.Security;
+import java.util.Properties;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import io.toast.tk.runtime.constant.Property;
+import io.toast.tk.runtime.utils.EncryptHelper;
+
+public class SmtpConfigProvider {
+
+	private static final Logger LOG = LogManager.getLogger(SmtpConfigProvider.class);
+
+	private String proxyURL;
+	private String proxyPort;
+	
+	private Properties properties;
+	
+	public static final String SMTP_PROPERTIES_FILE_NAME = "smtp.properties";
+	public static final String SMTP_PROPERTIES_FILE_STREAM_PATH = "/" + SMTP_PROPERTIES_FILE_NAME;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this hard-coded path-delimiter. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1075)